### DISCUSSION TITLE
How can I see the logs for a job?

### DISCUSSION THREAD

**Discussion by leo-guinan** [quirrel-dev/quirrel#713] [Q&A]
I've got my queue set up and I get it triggered. But It isn't doing what I want it to and I don't see any way to get access to the logs of the process. I assume that something is failing but I don't have any output to check to figure out exactly what is failing.

I feel like I'm missing something obvious, but for the life of me, I can't figure out what. :sweat_smile: 

Edit: I was missing something obvious. I had a typo in the path to my queue. But it ended up acting like it worked. Should it throw an error if there is a mismatch between the path and the URL used?

**Comment by Skn0tt** [comment]
Hi Leo! Jobs are executed inside of your application (not inside of Quirrel!), so you can access logs in the same way you access other logs.

For the typo: If you have a typo in a path, that causes the execution of the job to result in a 404, which triggers an "incident". On https://quirrel.dev, you'll be notified of this via e-mail. If you're running a self-hosted instance you can configure these to be sent to some HTTP endpoint, which can be configured using the `INCIDENT_RECEIVER_ENDPOINT` and `INCIDENT_RECEIVER_PASSPHRASE` env vars.

Sadly, there's no way for Quirrel to check for a mismatch between path and URL. If there was one, I'd be more than happy to just remove the `path` parameter altogether and infer it from the URL!
